pages/instructorQuestionManualGrading/instructorQuestionManualGradingNextInstanceQuestion.js

```diff
@@ -0,0 +1,30 @@
+const ERR = require('async-stacktrace');
+const express = require('express');
+const router = express.Router();
+const path = require('path');
+const debug = require('debug')('prairielearn:' + path.basename(__filename, '.js'));
+const {sqlDb, sqlLoader} = require('@prairielearn/prairielib');
+const sql = sqlLoader.loadSqlEquiv(__filename);
+
+router.get('/', (req, res, next) => {
+    const params = {
+        assessment_question_id: res.locals.assessment_question_id,
+        assessment_id: res.locals.assessment.id,
+    };
+    // Unmarked instance question df. Is last created submission of instance question AND has null graded_at value.
+    sqlDb.queryZeroOrOneRow(sql.get_next_unmarked_instance_question, params, (err, result) => {
```

Comment:
Terminology: let's try and standardize on `graded` rather than `marked` (this is partially a US-vs-UK-English thing).
```suggestion
    sqlDb.queryZeroOrOneRow(sql.get_next_ungraded_instance_question, params, (err, result) => {
```